pages/dkp/konvoy/2.0/image-builder/override-files/index.md

```diff
@@ -9,4 +9,36 @@ menuWeight: 75
 ---
 ## Override files
```

Comment:
My main concern initially was maintainability. For example, if something changes or is added in a new version of KIB that isn't in an older version, the overrides would not be valid for that version. These changes are definitely more 'static', but docs/engineering will need to make sure that they are maintained and updated according to any changes to KIB.